tests/e2e/commands_e2e/blockchain/deploy/deploy_tests.json

```diff
@@ -0,0 +1,23 @@
+{
+  "globalFlags": {
+    "local": true,
+    "skip-icm-deploy": true,
+    "skip-update-check": true,
+    "blockchainName": "testSubnet"
+  },
+  "happyPath": [
+    {
+      "name": "local_deploy",
+      "flags": {}
+    }
```

Comment:
we can set globalFlags in each testCases as well, which will override the globalFlags value